### PR TITLE
edits text for clarity

### PR DESCRIPTION
**Pull request by lwinfree**
Hi @roll! Here are my copyedits for livemark text. Thanks!

### CODE REVIEW

**Comment by lwinfree** [comment]
OK this is ready to go now :-)

File: README.md [hunk 1]
```diff
@@ -6,21 +6,21 @@
...
+> This software is in the early stages and is not well-tested
 
 Livemark is a static page generator that extends Markdown with interactive charts, tables, scripts, and more.
 
 ## Purpose
 
-- **Improved writing experience**: You can write data articles the way use usually do but with support of interactive charts, tables, live development server, and many more.
-- **Static site generator**: Livemark is fully predictable as it only operates with static files. It create an HTML file from your Mardown file.
+- **Improved writing experience**: Now your Markdown data articles can include interactive charts, tables, a live development server, and much more.
+- **Static site generator**: Livemark only operates with static files - it createa an HTML file from your Mardown file.
```
Comment (roll) [inline README.md:16]: Hi @lwinfree, I think it's a typo

Comment (lwinfree) [reply README.md]: good catch! I just fixed it - thanks!

Comment (roll) [reply README.md]: Thanks!

**Pull request merged**